libs/src/services/dataContracts/playlistFactoryClient.js

```diff
@@ -285,6 +288,14 @@ class PlaylistFactoryClient extends ContractClient {
       contractAddress
     )
   }
+
+  async isTrackInPlaylist (playlistId, trackId) {
+    const result = await this.getMethod('isTrackInPlaylist',
```

Comment:
nit: why the weird mix of async-await + promises?

Reply:
updated